.vsts-ci.yml

```diff
@@ -43,7 +45,7 @@ phases:
         export STORAGE_ACCOUNT_RESOURCE_ID=$(STORAGE_ACCOUNT_RESOURCE_ID)
         export TENANT_ID=$(TENANT_ID)
         pytest --numprocesses=70 tests/integration_tests
-      condition: and(succeeded(), or(startsWith(variables['Build.SourceBranchName'], 'release'), eq(variables['Build.SourceBranchName'], 'master')))
```

Comment:
is this for testing?

Reply:
this condition has been removed just for testing, yes.